Maps.md

```diff
@@ -0,0 +1,18 @@
+This is a general guide about Maps for all FullScreenShenanigans projects. This guide will explain the concept of Maps, Areas, and Locations within a game and how they're connected.
+
+### Table of Contents
+1. [MapsCreatr](#mapscreatr)
+
+# MapsCreatr
+
+MapsCreatr is a module that handles the storage of Maps, Areas, and Locations. 
+
+A Map is a collection of Areas and an Area is a space with predefined boundaries and Things. This is analogous to a Map being a city, while Areas could be different houses, streets, etc. Locations are entry points within an Area, where an entry point is where the player ends up after transitioning to a new Area.
+
+Generally in a FullScreenShenanigans project, Map information is stored in its own .js file, outlining all Areas and Locations within that map. When the game is started, MapsCreatr loads and stores all this information for later reference.
+
+# AreaSpawnr
+
+AreaSpawnr is a module that works with MapsCreatr to do the front-end displaying of Areas and the Things within them.
+
+When a new Area is entered by the player, AreaSpawnr uses the information from that Area's Map, stored in MapsCreatr, to place that Area's predefined Things (or PreThings) in the Area. This is what gives the actual visual representation of the Area. 
```

Comment:
"visual representation" is not what we're going for here. Same comment as above.